core/src/main/java/com/linecorp/armeria/client/ClientFactoryOption.java

```diff
@@ -75,8 +75,18 @@
      * The {@link Consumer} which can arbitrarily configure the {@link SslContextBuilder} that will be
      * applied to the SSL session.
      */
+    public static final ClientFactoryOption<Consumer<? super SslContextBuilder>> TLS_CUSTOMIZER =
```

Comment:
Just wanted to make sure that we don't have to use same list as we do in the server-side
`List<Consumer<? super SslContextBuilder>>`?

Reply:
I think it's fine as it is because it's more generic, and we do not expose TLS customizers in the server-side public API anyway.